internal/plugin/connectors/tcp/mssql/connection_details_test.go

```diff
@@ -14,18 +14,18 @@ var defaultConnectionDetails = &ConnectionDetails{
 	Password: "derp",
 	Host:     "0.0.0.0",
 	Port:     1234,
-	SSLMode:  "disable",
-	SSLOptions: map[string]string{
-		"sslrootcert": "foo",
-		"sslkey":      "bar",
-		"sslcert":     "foobar",
+	SSLParams: map[string]string{
+		"encrypt":                "true",
+		"trustServerCertificate": "true",
```

Comment:
`s/trustServerCertificate /trustservercertificate`, it must be all lowercase

Reply:
good catch - corrected